core/playwright.config.ts

```diff
@@ -39,7 +40,7 @@ export default defineConfig({
 	// max 30 seconds per test in CI
 	timeout: process.env.CI ? 30 * 1000 : 10 * 60 * 1000,
 	/* Reporter to use. See https://playwright.dev/docs/test-reporters */
-	reporter: process.env.CI ? "github" : "html",
+	reporter: process.env.CI ? "github" : "list",
```

Comment:
don't know about you but i hate it opening my browser every time. we could change this back